os/logm/Kconfig

```diff
@@ -5,6 +5,7 @@
 
 config LOGM
 	bool "Logger Module"
+	depends on !DISABLE_SIGNALS
```

Comment:
Right. logm also has a dependency on !CONFIG_BUILD_KERNEL because of task_create.
Could you please update it?

Reply:
Don't add this dependancy. We need to change "task_create" to "kernel_thread".

Reply:
It looks enough without modification I said in this commit.
Because I need to consider that logm task is made as normal or kernel thread.
If logm task is kernel thread, it will call kernel_thread instead of task_create.
I will update that later.